index.js

```diff
@@ -0,0 +1,135 @@
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { parse: gitDiffParser } = require('what-the-diff');
+const fs = require('fs-extra');
+const simpleGit = require('simple-git');
+const env = process.env;
+
+function matchExact(r, str) {
+  let match = str.match(r);
+  return match && str === match[0];
+}
+
+function fixNewLineEOF(b) {
+  // If file is empty
+  if (b.length < 1) {
+    return b;
+  }
+  // If file is not empty
+  b = b.replace(/[ \t\n]*$/, '\n');
+
+  if (b.length === 1) {
+    // if the remaining character is a whitespace
+    if (/[ \t\n]*$/.test(b)) {
+      return '';
+    } else {
+      return b + '\n';
+    }
+  }
+  return b;
+}
+
+async function getChangedFilesPaths(pull_request, octokit, owner, repo) {
+  const { data: pullRequestDiff } = await octokit.pulls.get({
+    owner,
+    repo,
+    pull_number: pull_request.number,
+    mediaType: {
+      format: 'diff'
+    }
+  });
+
+  const parsedDiff = gitDiffParser(pullRequestDiff);
+
+  const changedFilePaths = parsedDiff.map((e) => {
+    return e['newPath'].replace('b/', '');
+  });
+
+  return changedFilePaths;
+}
+
+async function run() {
+  const token = core.getInput('GH_TOKEN');
+  let ignorePaths = core.getInput('IGNORE_PATHS');
+  if (!ignorePaths) {
+    ignorePaths = [];
+  } else {
+    ignorePaths = ignorePaths.split(' ');
```

Comment:
This is not ideal - it means that paths can't have a space. Can you please use the same format used [here](https://github.com/gps/intellij-inspections#ignore_file_patterns)? You can look at the python code in that repo for a detailed example of how it is parsed.

Reply:
Done